102233 update travel claim copy to remove "30 days" language

PhilipDeFraties opened the pull request:
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary
* Adds updated translations for travel claim flow texts, designated with "--UPDATED" flags at end of keys
* Updated text provided by the wireframes linked in the issue

## Related issue(s)

https://github.com/orgs/department-of-veterans-affairs/projects/1323/views/35?pane=issue&itemId=95997463&issue=department-of-veterans-affairs%7Cva.gov-team%7C102233

## Testing done
Go through the travel claim flow and verify the language identified in the screenshots appear as expected with the "30 days" language removed. Be sure to check the error screens as well such as the missing uuid error by nav'ing to the start of the flow without the uuid in url (http://localhost:3001/my-health/appointment-travel-claim/), or by answering "No" to either of the travel questions.

## Screenshots

| Before | After |
|---------|---------|
| ![image](https://github.com/user-attachments/assets/7bdb4b55-45e9-4796-9c10-d40433e51be9) | ![image](https://github.com/user-attachments/assets/e246d7a2-e97f-451d-9c6e-c758f7d58cd7) |
 ![image](https://github.com/user-attachments/assets/eb441532-1378-4947-9ec5-3e409638760e) | ![image](https://github.com/user-attachments/assets/c9900360-536a-4532-952d-a2882453afb9) |
 ![image](https://github.com/user-attachments/assets/125949c5-6d00-4dd6-9927-af7572ecc642) | ![image](https://github.com/user-attachments/assets/fe415665-4152-4c2e-8d8a-7e9638983c4a) |
 ![image](https://github.com/user-attachments/assets/89b23709-5204-4aa3-97f6-50dbda525b29) | ![image](https://github.com/user-attachments/assets/26ef2026-f80c-41f1-a2d6-5b2e8ae7806d) |
![image](https://github.com/user-attachments/assets/ec22021a-31d9-4951-94bb-0b2ddb86e6d8) | ![image](https://github.com/user-attachments/assets/4b401678-e3c1-4d1c-bd92-312a611d2efc) |
![image](https://github.com/user-attachments/assets/e34863c6-4ca4-4bb7-96b2-56f44309fad7) | ![image](https://github.com/user-attachments/assets/7e3ea8e6-7b2a-49b8-92f1-6e1089d163ea) |


## What areas of the site does it impact?

Travel claims error messages and help texts

## Acceptance criteria

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

I'm still a bit unsure if this is the correct process after the discussion this morning, adding the new versions of the translations with the "--UPDATED" flag at the end of the keys seems to be in line with what is [indicated in the readme](https://github.com/orgs/department-of-veterans-affairs/projects/1323/views/35?pane=issue&itemId=95997463&issue=department-of-veterans-affairs%7Cva.gov-team%7C102233), but it seems that the [last time some of these values were updated](https://github.com/department-of-veterans-affairs/vets-website/pull/31826) both the utilized versions and their "--UPDATED" counterparts were added at the same time so I suspect that is what needs to happen here despite the wording in the readme, if this is the case I can get the subsequent commit added.

wullaski commented on src/applications/check-in/locales/en/translation.json:
```diff
@@ -436,10 +436,10 @@
-  "if-you-need-to-submit-receipts-for-other-expenses--helptext--UPDATE": "If you need to submit receipts for other expenses like tolls, meals, or lodging, you can’t file a claim now. But you can file your claim online, within 30 days, through the Beneficiary Travel Self Service System (BTSSS). You can also file by mail or in person using VA Form 10-3542.",
-  "if-you-traveled-by-bus-train-taxi-or-other-authorized--help-text--UPDATE": "<0>If you traveled by bus, train, taxi, or other authorized public transportation, you can’t file a claim in this tool right now.</0> But you can file your claim online, within 30 days, through the Beneficiary Travel Self Service System (BTSSS). You can also file by mail or in person using VA Form 10-3542.",
-  "if-you-traveled-from-a-different-address-you--helptext--UPDATE": "<0>If you traveled from a different address, you can’t file a claim in this tool right now.</0> But you can file your claim online, within 30 days, through the Beneficiary Travel Self Service System (BTSSS). You can also file by mail or in person using VA Form 10-3542.",
+  "if-you-need-submit-receipts-other-expenses--UPDATE": "<0>If you need to submit receipts for other expenses like tolls, meals, or lodging, you can’t file a claim in this tool right now.</0> But you can file your claim online through the Beneficiary Travel Self Service System (BTSSS). You can also file by mail or in person using VA Form 10-3542.",
```
These keys will need to change in order to be picked up by Lori's script, so something like --update-30-days.

wullaski commented on src/applications/check-in/locales/en/translation.json:
```diff
@@ -450,5 +450,10 @@
-}
\ No newline at end of file
+  "how-to-file-other-types-of-claims": "How to file other types of claims",
+  "we-cant-file-a-claim--UPDATE": "We can’t file a travel reimbursement claim for you in this tool right now. But you can still file through the Beneficiary Travel Self Service System (BTSSS).",
```
If you left the originals in they can be removed from this file and the other translations files.

wullaski commented on src/applications/check-in/locales/en/translation.json:
if you change any keys do a find and replace on the old names in the check-in folder.

wullaski reviewed (changes requested):
Let me know if the comments make sense. Happy to sync up with you and go over the process.

PhilipDeFraties committed:
update travel claim translations with new versions flagged with --v2 to remove "30 days" language

delete old key/value pairs from en and tl translation files, will add new versions once translations are provided

update key references to use new keys flagged with --v2

PhilipDeFraties committed:
update checkin spec assertions with new travel reimbursement text

cgednalske commented:
Confirming this all looks good 👍 @PhilipDeFraties @wullaski

wullaski reviewed (approved):
LGTM!
